configure.ac

```diff
@@ -305,6 +305,23 @@ if test "$build_docs" -eq 1; then
 fi
 
 
+# Add assert() usage
+
+AC_ARG_ENABLE(assertions,
+              AS_HELP_STRING([--enable-assertions],
+                             [Turn on assertions checks (undefine NDEBUG)]),
+[
+  if test "${enableval}" = "yes"; then
+      assertions='-UNDEBUG'
+      MODSEC_EXTRA_CFLAGS="$MODSEC_EXTRA_CFLAGS $assertions"
```

Comment:
Why do you extend this CFLAG here? Wouldn't be a better place in line [849](https://github.com/owasp-modsecurity/ModSecurity/pull/3120/files#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810R849)?